manifests/c/CrystalDewWorld/CrystalDiskInfo/KureiKeiEdition/8.12.3/CrystalDewWorld.CrystalDiskInfo.KureiKeiEdition.installer.yaml

```
@@ -0,0 +1,22 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CrystalDewWorld.CrystalDiskInfo.KureiKeiEdition
+PackageVersion: 8.12.3
+MinimumOSVersion: 10.0.0.0
+Platform:
+- Windows.Desktop
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://osdn.net/frs/redir.php?m=bfsu&f=crystaldiskinfo%2F75470%2FCrystalDiskInfo8_12_3KureiKei.exe
```

```suggestion
  InstallerUrl: https://osdn.net/frs/redir.php?m=auto&f=crystaldiskinfo%2F75470%2FCrystalDiskInfo8_12_3KureiKei.exe
```
So that it redirects to proper server.